### PR TITLE
Adding 3 Guides on advanced assets creation

### PR DESCRIPTION
**Pull request by Dytser**
Adding 3 guides
Material Slots, Character Customisation Sliders (shapekeys, likely will make one for heads etc), And Jiggle Physics/phys bones.

### CODE REVIEW

File: docs/asset-swapping/Advanced Asset Creation/Character Customisation Colors.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Guide written by: Dytser
```
Comment (localcc) [inline docs/asset-swapping/Advanced Asset Creation/Character Customisation Colors.md]: Maybe make the `Guide written by` thing bold, as well as the other line

File: docs/asset-swapping/Advanced Asset Creation/Jiggle Physics.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+The amount of weight applied depends entirely on what you are trying to add physics to.
+#### Bodyparts
+For things that are attached to the body such as breasts, butt, thigh, stomach, ears etc, You likely don't want to overdo it, I recommend starting small and increase from there if you want, For breast a good recommended weight would be 0.5 for example.
+
+#### Detached Parts
+For things not attached to a characters body (hair, clothing tassels, etc) you generally would like to go stronger as having the part be "static" wouldnt look realistic, And unlike body fat physics which generally is more for secondary motion detached parts are more or less "made" of physics.
+
+For example having a keychain attached to a weapon with a figurine on the other end of the chain. we expect this figure to move and adhere to physics entirely rather than only partially like we do a body.
+
+### exporting
```
Comment (localcc) [inline docs/asset-swapping/Advanced Asset Creation/Jiggle Physics.md]: Capitalize

**Review by localcc — CHANGES_REQUESTED** [review]

**Comment by Dytser** [comment]
All very good points, Ill sort those out.

File: docs/asset-swapping/Advanced Asset Creation/Jiggle Physics.md (hunk 2 above)
Comment (Dytser) [reply docs/asset-swapping/Advanced Asset Creation/Jiggle Physics.md]: Fixed!

File: docs/asset-swapping/Advanced Asset Creation/Character Customisation Colors.md (hunk 1 above)
Comment (Dytser) [reply docs/asset-swapping/Advanced Asset Creation/Character Customisation Colors.md]: Sorted!
Did the same with the other two files!

**Review by localcc — APPROVED** [review]

**Pull request merged**